scipy/stats/tests/test_stats.py

```diff
@@ -3958,13 +3975,29 @@ def test_ks1samp_allpaths(self):
         ])
         FuncData(kolmogn, dataset, (0, 1, 2), 3).check(dtypes=[int, float, bool])
 
+    @pytest.mark.parametrize("alternative, x67val, ref_location, ref_sign",
+                             [('greater', 0.5, 0.5, +1),
+                              ('less', 0.7, 0.7, -1),
+                              ('two-sided', 0.5, 0.5, +1),
+                              ('two-sided', 0.7, 0.7, -1)])
+    def test_location_sign(self, alternative, x67val, ref_location, ref_sign):
+        # Test that location and sign corresponding with statistic are as
+        # expected. (Test is designed to be easy to predict.)
+        x = np.arange(1.0, 11, dtype=np.float64) / 10.0
+        x[5:7] = x67val
+        res = stats.ks_1samp(x, stats.uniform.cdf, alternative=alternative)
+        res_list = [res.statistic, res.statistic_location, res.statistic_sign]
+        expected_list = [0.2, ref_location, ref_sign]
+        assert_array_almost_equal(np.array(res_list), np.array(expected_list))
```

Comment:
```suggestion
        assert_array_almost_equal(res_list, expected_list)
```
This is not needed here or elsewhere.